packages/ui/src/ui/containers/AuthenticationTokensModal/AuthenticationTokensModal.tsx

```diff
@@ -0,0 +1,397 @@
+import cn from 'bem-cn-lite';
+import moment from 'moment';
+import * as React from 'react';
+import {FC, useMemo, useState} from 'react';
+import {Alert, Button, Dialog, TextInput} from '@gravity-ui/uikit';
+import DataTable from '@gravity-ui/react-data-table';
+import {FormApi} from '@gravity-ui/dialog-fields';
+import truncate from 'lodash/truncate';
+import Modal from '../../components/Modal/Modal';
+import {YTDFDialog} from '../../components/Dialog/Dialog';
+import AuthenticationTokensNothingFound from './authentication-tokens-nothing-found.svg';
+import {
+    authenticationTokensClearCredentials,
+    authenticationTokensCreateToken,
+    authenticationTokensGetList,
+    authenticationTokensRevokeToken,
+    authenticationTokensSetCredentials,
+    clearAuthenticationTokensFilter,
+    closeAuthenticationTokensModal,
+    filterAuthenticationTokens,
+} from '../../store/actions/authentication-tokens';
+import {useThunkDispatch} from '../../store/thunkDispatch';
+import Error from '../../components/Error/Error';
+import {useDispatch, useSelector} from 'react-redux';
+import {
+    AuthenticationToken,
+    getAuthenticationTokens,
+    getAuthenticationTokensCredentials,
+    isAuthenticationTokensModalOpened,
+} from '../../store/selectors/authentication-tokens';
+import Icon from '../../components/Icon/Icon';
+import {YTError} from '../../../@types/types';
+import ClipboardButton from '../../components/ClipboardButton/ClipboardButton';
+
+import './AuthenticationTokensModal.scss';
+
+const block = cn('authentication-tokens');
+
+const renderNoContent = () => {
+    return (
+        <div className={block('no-content')}>
+            <img src={AuthenticationTokensNothingFound} />
+            <p>No tokens found</p>
+        </div>
+    );
+};
+
+const AuthenticationGenerateTokenFormSection: FC<DefaultProps> = ({setSection}) => {
```

Comment:
It is better to have `onClose` instead of `setSection`. The component should not know anything about sections.